salt/modules/yumpkg.py

```diff
@@ -715,7 +795,10 @@ def _parse_output(output, strict=False):
             )
     # Really old version of yum; does not even have --showduplicates option
     if yum_version and yum_version < _LooseVersion('3.2.13'):
-        cmd_prefix = ['yum', '--quiet', 'list']
+        cmd_prefix = ['yum', '--quiet']
+        if cacheonly:
+            yum_prefix.append('-C')
```

Comment:
Yeah, typo. Fixed it, as well as lint, and rebased the changes into their respective commits.